SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -795,18 +800,25 @@ - (void)sdl_hmiStatusDidChange:(SDLRPCNotificationNotification *)notification {
     SDLHMILevel oldHMILevel = self.hmiLevel;
     self.hmiLevel = hmiStatusNotification.hmiLevel;
 
-    SDLAudioStreamingState oldStreamingState = self.audioStreamingState;
+    SDLAudioStreamingState oldAudioStreamingState = self.audioStreamingState;
     self.audioStreamingState = hmiStatusNotification.audioStreamingState;
 
+    SDLVideoStreamingState oldVideoStreamingState = self.videoStreamingState;
+    self.videoStreamingState = hmiStatusNotification.videoStreamingState;
+
     SDLSystemContext oldSystemContext = self.systemContext;
     self.systemContext = hmiStatusNotification.systemContext;
 
     if (![oldHMILevel isEqualToEnum:self.hmiLevel]) {
         SDLLogD(@"HMI level changed from %@ to %@", oldHMILevel, self.hmiLevel);
     }
 
-    if (![oldStreamingState isEqualToEnum:self.audioStreamingState]) {
-        SDLLogD(@"Audio streaming state changed from %@ to %@", oldStreamingState, self.audioStreamingState);
+    if (![oldAudioStreamingState isEqualToEnum:self.audioStreamingState]) {
+        SDLLogD(@"Audio streaming state changed from %@ to %@", oldAudioStreamingState, self.audioStreamingState);
+    }
+
+    if (![oldAudioStreamingState isEqualToEnum:self.videoStreamingState]) {
```

Comment:
Should be comparing `oldVideoStreamingState` to `self.videoStreamingState` instead of `oldAudioStreamingState`. Also, the log prints out `oldAudioStreamingState` instead of `oldVideoStreamingState`

Reply:
Fixed

Reply:
Fixed